test/suites/common-container-chains/dynamic-params/test-relay-network.ts

```diff
@@ -1,7 +1,10 @@
+import "@moonbeam-network/api-augment";
```

Comment:
why not import the tanssi-api¿

Reply:
Ive cast the types so we dont need this anymore anyway